File: examples/charge.ts
import { drip } from '@drip-sdk/node';

async function main() {
  const customer = await drip.createCustomer({ externalCustomerId: 'user_123' });

  await drip.charge({
    customerId: customer.id,
    meter: 'tokens',
    quantity: 1847,
  });
}

void main();

File: python/examples/charge.py
from drip import drip


def main() -> None:
    customer = drip.create_customer(external_customer_id="user_123")

    drip.charge(
        customer_id=customer.id,
        meter="tokens",
        quantity=1847,
    )


if __name__ == "__main__":
    main()
